src/main/java/com/github/maxopoly/MemeMana/command/CmdManaGive.java

```diff
@@ -0,0 +1,50 @@
+package com.github.maxopoly.MemeMana.command;
+
+import vg.civcraft.mc.civmodcore.command.PlayerCommand;
+import vg.civcraft.mc.namelayer.NameAPI;
+import com.devotedmc.ExilePearl.ExilePearlPlugin;
+import com.devotedmc.ExilePearl.ExilePearl;
+import com.devotedmc.ExilePearl.PearlType;
+import com.github.maxopoly.MemeMana.model.MemeManaPouch;
+import com.github.maxopoly.MemeMana.model.MemeManaOwner;
+import com.github.maxopoly.MemeMana.model.ManaGainStat;
+import com.github.maxopoly.MemeMana.MemeManaPlayerOwner;
+import com.github.maxopoly.MemeMana.MemeManaPlugin;
+import org.bukkit.entity.Player;
+import org.bukkit.command.CommandSender;
+import java.util.UUID;
+import java.util.List;
+import java.util.LinkedList;
+import java.util.function.IntFunction;
+
+public class CmdManaGive extends PlayerCommand {
+	public CmdManaGive(String name) {
+		super(name);
+		setIdentifier("managive");
+		setDescription("Give a player some mana");
+		setUsage("/managive Player Amount");
+		setArguments(2,2);
```

Comment:
Add a short alias for all the commands that people can easily use like mmg (meme mana give)

Reply:
How do I do this? Just another `setIdentifier("mmg");`?